src/api/cart.ts

```diff
@@ -0,0 +1,53 @@
+import { AUTH_HEADER } from './auth';
+import { END_POINT } from './endpoints';
+import fetcher from './fetcher';
+
+import { CartItemInfo } from '@/types/cartItem';
+
+interface AddCartItemProp {
+  productId: number;
+  quantity?: number;
+}
+
+type MutationResponse = Record<'status', number>;
+
+export const fetchCartItems = async (): Promise<CartItemInfo[]> => {
+  const response = await fetcher.get({
+    url: `${END_POINT.cartItems}`,
```

Comment:
```suggestion
    url: `${END_POINT.cartItems}?size=2000`,
```
현재 장바구니 아이템이 20개까지밖에 들어가지 않는 문제가 있어요!
cart_items도 product와 같이 page, size, sort를 받기 때문에
위와 같이 코드를 수정하면 될 것 같아요 👍